kubernetes/ansible/roles/sunbird-monitoring/templates/dp_prometheus-adapter.yaml

```diff
@@ -0,0 +1,209 @@
+# Default values for k8s-prometheus-adapter..
```

Comment:
@sowmya-dixit you don't have to add all variables here. Put only the changed ones, so that it'll be easier to keep track of.

Reply:
@sowmya-dixit how are we applying this change to the helm installation? because the template name is dp_prometheus-adapter.yaml and the chart name is prometheus-adapter.yaml

Reply:
File is used as **dp_prometheus-adapter.yaml** in helm installation under tasks.main.yaml

Reply:
@rjshrjndrn I have updated yaml file to have only the changes ones.